config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/EncassEntityBuilder.java

```diff
@@ -72,7 +72,11 @@ private Entity buildEncassEntity(Bundle bundle, String name, Encass encass, Docu
                 buildResults(encass, document)
         );
 
-        buildAndAppendPropertiesElement(ImmutableMap.of(PALETTE_FOLDER, INTERNAL_ASSERTIONS), document, encassAssertionElement);
+        buildAndAppendPropertiesElement(
+                encass.getProperties() == null ?
+                        ImmutableMap.of(PALETTE_FOLDER, INTERNAL_ASSERTIONS) :
```

Comment:
I believe you have to check if this property is present in a possible existing map and add if not.